browser/src/control/Control.ESignature.ts

```diff
@@ -100,12 +110,96 @@ namespace cool {
 			'serpro-id-advanced-signature': 'SerproID',
 		};
 
+		// Country code to name map
+		static countryNames: { [name: string]: string } = undefined;
+
 		constructor(url: string, clientId: string) {
 			this.url = url;
 			this.clientId = clientId;
 
 			app.map.on('commandvalues', this.onCommandValues.bind(this));
 			app.map.on('commandresult', this.onCommandResult.bind(this));
+
+			if (!ESignature.countryNames) {
+				ESignature.countryNames = {
+					// Generated by scripts/countries.py
```

Comment:
I asked for input from @timar here: either we bundle an npm package (it has an additional dependency) that has these strings is painful external JSON files (but then no duplication here) or we have this here and we avoid the bloat of a new dependency. His suggestion was to have this inline, but do it with a script. I like this approach, because then this avoids the political issues with country names: we just take the names as-is from the ISO data.

Reply:
I think i18n-iso-countries has done some (sensible) processing of the ISO data, because notoriously 3166 has a short(!) name of "United Kingdom of Great Britain and Northern Ireland" for the UK :-)

"AX" is unusual choice for inclusion, but I guess that's eID's decision

Reply:
There are a few others, like https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#TR says "TR" is Türkiye, but this package sticks to Turkey.